db/migrate/20161219225847_init_schema.rb

```diff
@@ -89,7 +88,7 @@ def up
       t.integer  "role"
       t.string   "otp_secret_key",                limit: 255
       t.integer  "second_factor_attempts_count",              default: 0
-      t.string   "phone",                         limit: 255
+      t.string   "phone_plain",                   limit: 255
```

Comment:
we might need to include more squash in this, since `phone_plain` should not be present anymore.